tree/dataframe/inc/ROOT/RDF/Utils.hxx

```diff
@@ -203,6 +204,17 @@ bool IsInternalColumn(std::string_view colName);
 /// Get optimal column width for printing a table given the names and the desired minimal space between columns
 unsigned int GetColumnWidth(const std::vector<std::string>& names, const unsigned int minColumnSpace = 8u);
 
+#if __cplusplus < 201703L
+   // assume the typical 64 bytes
+   static constexpr std::size_t kCacheLineSize = 64;
+#else
+   inline constexpr std::size_t kCacheLineSize = std::hardware_destructive_interference_size;
+#endif
+
+// Stepping these many values brings you to a new cache line. Useful to avoid false sharing.
+static constexpr std::size_t kCacheLineLong64Step = kCacheLineSize / sizeof(Long64_t);
+static constexpr std::size_t kCacheLineIntStep = kCacheLineSize / sizeof(int);
```

Comment:
```suggestion
template <typename T>
constexpr std::size_t CacheLineStep() {
   return RDFInternal::kCacheLineSize + sizeof(T) - 1) / sizeof(T);
}
```
and use that also to determine the value of `RDefine::kCacheLineStep`?

Reply:
nice, thanks!

Reply:
done